vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/BatchRegistrationDialog.java

```diff
@@ -22,7 +20,7 @@
  *
  * <p>Component to register {@link Sample} with their associated metadata information</p>
  */
-public class BatchRegistrationDialog extends Dialog {
+public class BatchRegistrationDialog extends DialogWindow {
```

Comment:
div tag needed here?

Reply:
Good point! I tried it and it leads to Javascript errors and the Component being rendered below everything (not in a Dialog Window). So my take is that a Dialog cannot/should not be a Div.